pages/_app.tsx

```diff
@@ -10,16 +10,45 @@ import type { AppProps } from 'next/app';
 import { ChakraProvider } from '@chakra-ui/react';
 import theme from '../theme';
 import '../styles/Mobile.css';
+import '../styles/globals.css';
 import Head from 'next/head';
 import { Web3Provider } from '../contexts/Web3Context';
 import Layout from '../components/Layout';
 
 function MyApp({ Component, pageProps }: AppProps) {
+
+  // Production MixPanel -> Bankless DAO
+  const MIX_PANEL_ID = '0715c4ae0cba2709f3d752f073ef2824'
+  const DEBUG = false
+
+  // Development MixPanel -> Bankless DAO Dev
+  // const MIX_PANEL_ID = '63978c515162fcb62e8a7da786c5a17b'
+  // const DEBUG = true
+
+  console.log('TEST LOG. MY APP IS RUNNING')
```

Comment:
Do you want this console.log in?

Reply:
@singularityhacker

Reply:
It's fine. Not enough of a negative impact to deal with right now. We merely want an analysis on the site so we can compare it to the new site when we go live in two months.